src/Controller/EasyAdminController.php

```diff
@@ -3,6 +3,7 @@
 namespace AlterPHP\EasyAdminExtensionBundle\Controller;
 
 use AlterPHP\EasyAdminExtensionBundle\Security\AdminAuthorizationChecker;
+use AlterPHP\EasyAdminExtensionBundle\Controller\AdminExtensionControllerTrait;
 use EasyCorp\Bundle\EasyAdminBundle\Controller\EasyAdminController as BaseEasyAdminControler;
 use EasyCorp\Bundle\EasyAdminBundle\Event\EasyAdminEvents;
```

Comment:
Some `use` statements are useless now. Please remove them, or use.

Reply:
Done in last commit